assets/src/design-system/components/popup/index.js

```diff
@@ -38,6 +38,7 @@ const Container = styled.div.attrs(
     },
   })
 )`
+  /*! @noflip */
```

Comment:
what's this for?

Reply:
This is copying in a fix that pascal made when we were working on the animation popup and we noticed popup location doesn't flip in wordpress. https://github.com/google/web-stories-wp/pull/5618

Reply:
oh v cool. til